apis/pipeline/pipeline.yaml

```diff
@@ -4665,7 +4704,7 @@ components:
                         - nodePools
                     properties:
                         authConfigMap:
```

Comment:
This could also be something like `authConfig` or `clusterAuth` or `clusterAuthConfig`

Reply:
I also changed this `authConfig`